### ISSUE TITLE
Contacts Status (Stories images, not connection status)

### ISSUE THREAD

**Issue by wazowski78** [danielcardeenas/sulla#542]
I wonder if it is possible to retrieve the Status images currently available when login in in the session. I mean Status images, not the status of the connection. Thanks a lot.

**Comment by danielcardeenas** [comment]
You mean the stories?

**Comment by wazowski78** [comment]
Yes, I mean stories, in WhatsApp are called status, but I think Stories is a better name for them. Great, I will wait for it them.
